data/rulesets.js

```diff
@@ -256,7 +262,8 @@ exports.BattleFormats = {
 		},
 	},
 	hoennpokedex: {
-		effectType: 'Rule',
+		effectType: 'Banlist',
```

Comment:
Okay, I finally understand now.

I think `ValidatorRule` is a clearer name here. Otherwise, no objection.